<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_coffee_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>651</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="id_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>651</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить данные по id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить изменения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_coffee">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_sorts">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Название кофе:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Название сорта:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>798</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Степень обжарки:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roast_degree">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="status">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>230</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Молотый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>В зернах</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="taste">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вкус:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Состояние:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="price">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Цена:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>380</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Объем:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="volume">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
